### DISCUSSION TITLE
x-ray core的telegram群，入群验证也太难了吧。。。根本答不对

### DISCUSSION THREAD

**Discussion by Kanon1982** [XTLS/Xray-core#3487] [General]
x-ray core的telegram群，入群验证也太难了吧。。。根本答不对
入群问题居然是：图中，哪个碳原子是手性碳原子。。。
这也太特么难了吧

**Comment by RPRX** [comment]
~~虽然问题有点魔幻，但是 Xray 能拼成 x-ray，答不对也很正常~~

~~要不开卖邀请码十刀一位吧~~ 其实等段时间就能重新答题了

**Reply by Kanon1982** [reply]
誒。。。吃了没文化的亏啊233333333333

**Reply by sky22333** [reply]
截图发给AI作答，入群验证不是考你的知识，是考你善用搜索和排查问题的能力

**Reply by Kanon1982** [reply]
关键是，我发给ai作答，ai回答的有很大概率是不正确的

**Reply by yrjjun** [reply]
> 截图发给AI作答，入群验证不是考你的知识，是考你善用搜索和排查问题的能力

我发给ai，ai也答不对，然后搜索就到这里了

**Comment by QwQIsHidden** [comment]
那个题目真的好难啊
还考我Minecraft的知识
我寻思着xray和mc有啥关系（

**Reply by sky22333** [reply]
截图发给AI作答，入群验证不是考你的知识，是考你善用搜索和排查问题的能力

**Reply by yrjjun** [reply]
> 截图发给AI作答，入群验证不是考你的知识，是考你善用搜索和排查问题的能力

我发给ai，ai也答不对，然后搜索就到这里了

**Reply by Q7DF1** [reply]
> 截图发给AI作答，入群验证不是考你的知识，是考你善用搜索和排查问题的能力

AI搞不明白，这玩意儿真得有点懂这块知识并且实操才能答对吧

**Comment by zzy9001** [comment]
实际的答题冷却是一个小时么，后续不断尝试加群都是提示not accessible。

**Comment by ludb68** [comment]
在Xray中，使用Reality且设置TLS指纹为safari时，对其JA4指纹sha384后，前六位是 
这个不是选 da0c01 么
![{0AE11A10-7E44-40FF-B60C-2FFC0DD1EF90}](https://github.com/user-attachments/assets/1f0fd01c-9d0e-4fd5-9d86-808cebf18ff7)

![{BC28E8A1-7FF8-4A9B-B1B5-940CEA25C92A}](https://github.com/user-attachments/assets/a9e4a518-13d6-4db6-8154-3004d4c811e2)

**Reply by ludb68** [reply]
选了这个没通过
疑惑

**Reply by xqzr** [reply]
有没有可能不含 `_`

**Reply by janesle** [reply]
有一个Q群讨论：939989248

**Comment by janesle** [comment]
没人回复啊，有一个Q群讨论：939989248

**Comment by LongLights** [comment]
<img width="1280" height="272" alt="10704bdfa6da7cf3e0a0a3d5754b31d7" src="https://github.com/user-attachments/assets/47c8708d-adca-4045-8e86-835cbc0a9449" />


同过不去，我sha4后得到的结果没有对应选项😢

**Comment by Gravifer** [comment]
我为什么要睡一觉醒来刚开机就做验证题。。。 zz vscode的hex编辑器硬控我到超时 难过了有点

**Reply by Gravifer** [reply]
5小时真有点狠了 能不能降到3h啊

**Comment by Q7DF1** [comment]
真的很难，考我一个steam 游戏社区相关内容，题都有点难读懂

**Comment by Q7DF1** [comment]
> x-ray core的telegram群，入群验证也太难了吧。。。根本答不对
> 入群问题居然是：图中，哪个碳原子是手性碳原子。。。
> 这也太特么难了吧

你别说要是是这种，我感觉我会，听上去像高中化学知识
